continuous_integration/install.sh

```diff
@@ -75,4 +75,8 @@ if [ -z "$NUMPY_VERSION" ]; then
     mv setup.cfg{.new,}
 fi
 
+if [ ! -z "$DILL_VERSION" ]; then
```

Comment:
Does this mean the pickle imports can be removed and replaced with dill imports?